package.json

```diff
@@ -176,6 +178,16 @@
                 "title": "Delete...",
                 "category": "Azure App Service"
             },
+            {
+                "command": "appService.appSettings.Download",
+                "title": "Download Remote Settings",
+                "category": "Azure App Service"
+            },
+            {
+                "command": "appService.appSettings.Upload",
+                "title": "Upload Local Settings",
```

Comment:
Both of these commands should have an ellipses since they ask for more information